### PR TITLE
feat: Adding member count logic to snapshot implementation

### PR DESCRIPTION
**Pull request by rossneilson**
Adding to snapshot lock and withdraw token implementation as currently it always stays at 1

### CODE REVIEW

File: contracts/erc20guild/implementations/SnapshotERC20Guild.sol [hunk 1]
```diff
@@ -79,6 +79,7 @@ contract SnapshotERC20Guild is ERC20Guild {
     // @dev Lock tokens in the guild to be used as voting power
     // @param tokenAmount The amount of tokens to be locked
     function lockTokens(uint256 tokenAmount) external virtual override {
+        if (tokensLocked[msg.sender].amount == 0) totalMembers = totalMembers.add(1);
```
Comment (AugustoL) [inline contracts/erc20guild/implementations/SnapshotERC20Guild.sol:82]: add linebreak for if/else

**Review by AugustoL — CHANGES_REQUESTED** [review]

File: contracts/erc20guild/implementations/SnapshotERC20Guild.sol (hunk 1 above)
Comment (AugustoL) [reply contracts/erc20guild/implementations/SnapshotERC20Guild.sol:82]: Also, do we have tests for this contract? if so add the check in the tests

Comment (rossneilson) [reply contracts/erc20guild/implementations/SnapshotERC20Guild.sol:82]: The erc20 guild implementation that you did doesnt have line breaks, should I add there too? Its not auto adding them.
And will add a test or two

Comment (AugustoL) [reply contracts/erc20guild/implementations/SnapshotERC20Guild.sol:82]: hmmm, I though we were requesting line breaks, I always use them. ok leave it then

**Review by AugustoL — APPROVED** [review]

**Pull request merged**